core/src/main/java/com/netflix/conductor/core/utils/ExternalPayloadStorageUtils.java

```diff
@@ -144,50 +144,38 @@ public ExternalPayloadStorageUtils(
             } else if (payloadSize > threshold * 1024) {
                 switch (payloadType) {
                     case TASK_INPUT:
-                        ((TaskModel) entity).setInputData(new HashMap<>());
-                        ((TaskModel) entity)
-                                .setExternalInputPayloadStoragePath(
-                                        uploadHelper(
-                                                payloadBytes, payloadSize, PayloadType.TASK_INPUT));
+                        String externalInputPayloadStoragePath1 =
```

Comment:
why the name of `externalInputPayloadStoragePath1`? I meant for the `1` postfix

Reply:
Corrected.